rollup-plugin-docz-starter.ts

```diff
@@ -7,7 +7,7 @@ function doczStarter() {
 
     return {
         name: 'docz-starter',
-        generateBundle: function() {
+        generateBundle: function () {
```

Comment:
Personally, I find this style *fugly*, but I have no argument of substance against it, and I don't feel like personal preference should matter all that much when it comes to setting conventions. Though honestly, I think the argument for the space is vapour thin as well ([link](https://prettier.io/blog/2020/03/21/2.0.0.html#always-add-a-space-after-the-function-keyword-3903httpsgithubcomprettierprettierpull3903-by-j-f1httpsgithubcomj-f1-josephfrazierhttpsgithubcomjosephfrazier-sosukesuzukihttpsgithubcomsosukesuzuki-thorn0httpsgithubcomthorn0-7516httpsgithubcomprettierprettierpull7516-by-bakkothttpsgithubcombakkot)).

Here's the buried lede: there doesn't seem to be an option for it, so we probably can't change it even if we wanted to.

Also, in all of the lime-elements code-base, this is the *single* instance we use this form 😄